src/scene/shader-lib/chunks/lit/frag/clusteredLight.js

```diff
@@ -543,45 +543,45 @@ void evaluateLight(ClusterLightData light) {
                 
                 // specular
                 #ifdef LIT_SPECULAR_FRESNEL
-                    dSpecularLight += getLightSpecular(halfDir) * dAtten * light.color * dAtten3 * getFresnel(dot(dViewDirW, halfDir), dSpecularity);
+                    dSpecularLight += getLightSpecular(halfDir, dReflDirW, worldNormal, gloss) * dAtten * light.color * dAtten3 * getFresnel(dot(dViewDirW, halfDir), gloss, specularity, iridescence);
```

Comment:
should we pass dReflDirW and ccReflDirW (and similar if any) as function arguments instead of using global, to limit changes to chunk in the future? Or are we ok to use globals for temp variables.